corehq/motech/dhis2/dhis2_config.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from couchdbkit.ext.django.schema import DocumentSchema, ListProperty, StringProperty, SchemaProperty, \
+    SchemaListProperty
+
+from corehq.motech.dhis2.const import DHIS2_EVENT_STATUSES, DHIS2_EVENT_STATUS_COMPLETED
+from corehq.motech.value_source import ValueSource
+
+
+class FormQuestion(ValueSource):
```

Comment:
This definition is duplicated in corehq/motech/openmrs/openmrs_config.py. I'd move *all* ValueSource subclasses into motech/value_source.py, including `CaseProperty`, `CasePropertyMap`, `ConstantString`, etc. It'll logically keep them together. (And if/when we integrate case properties with DHIS2 Tracked Entities, we won't have to move more stuff again.)

(The next DHIS2 integration will map case properties to Tracked Entities, so we'll need to share them with OpenMRS too.)